lib/mindwendel_web/live/idea_live/index_component.html.heex

```diff
@@ -37,27 +48,26 @@
           <% end %>
 
           <%= unless idea.link do %>
-            <%= text_to_html(idea.body) %>
+            <%= raw idea.body %>
           <% end %>
 
           <%= if idea.link do %>
-            <%= link to: idea.link.url do %>
-              <%= text_to_html(idea.body) %>
-            <% end %>
+            <.link href={idea.link.url}>
+              <%= raw idea.body %>
```

Comment:
`text_to_html ` is no longer available

Reply:
@JannikStreek but this right now allows for XSS, and also doesn't insert the line breaks as text_to_html did. txt_to_html has escaped HTML elements by default.

I guess there is some other function now that does this, but I think I'd rather than using `raw` have the text be escaped.

Reply:
ah good to know. I just threw out views completely and it still works 😅 , so close to finishing this :)